### ISSUE TITLE
cadence charts doesn't work with new versions

### ISSUE THREAD

**Issue by longquanzheng** [banzaicloud/banzai-charts#865]
I got errors when starting pods:
```
2019/06/18 20:17:25 Setting up version tables
2019/06/18 20:17:25 Cannot add already existing table "schema_version" to keyspace "cadence"
+ /cadence/cadence-cassandra-tool --ep my-cadence-2-cassandra -k cadence update-schema -d /cadence/schema/cassandra/cadence/versioned
2019/06/18 20:17:25 UpdateSchemeTask started, config=&{DBName: TargetVersion: SchemaDir:/cadence/schema/cassandra/cadence/versioned IsDryRun:false}
2019/06/18 20:17:25 error listing schema dir:version dir not found for target version
```

As I tried to use newer version of cadence image:
```
    repository: ubercadence/server
    tag: 0.5.9
```

**Comment by sagikazarmark** [comment]
Thanks for reporting the issue. We are looking into it.

**Comment by sagikazarmark** [comment]
Fixed in #866
